judgments/utils.py

```diff
@@ -203,13 +203,12 @@ def parse_date_parameter(params, param_name, default_to_last=False):
         raise ValueError(gettext("search.errors.missing_year_detail"))
 
 
-def get_document_by_uri(document_uri: str) -> Judgment:
+def get_document_by_uri(document_uri: str) -> Document:
     api_client = MarklogicApiClient(
         host=settings.MARKLOGIC_HOST,
         username=settings.MARKLOGIC_USER,
         password=settings.MARKLOGIC_PASSWORD,
         use_https=settings.MARKLOGIC_USE_HTTPS,
     )
 
-    # raises a JudgmentNotFoundError if it doesn't exist
-    return Judgment(document_uri, api_client)
+    return Document(document_uri, api_client)
```

Comment:
I think this is still really a `Judgment`, since we don't have any way of interpreting different document types right now.

Reply:
Not quite sure what you mean, will reach out on slack